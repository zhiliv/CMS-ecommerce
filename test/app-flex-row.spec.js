import { shallowMount } from '@vue/test-utils'
import flexRow from '@/components/app/flex-row/flex-row.vue'

describe('app-flex-row.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(flexRow, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        classes: 'simple',
        styles: 'background: white',
        id: 'indent',
        dFlex: true,
        dInlineFlex: true,
        flexRow: true,
        dSmFlex: true,
        dSmInlineFlex: true,
        dMdFlex: true,
        dMdInlineFlex: true,
        dLgFlex: true,
        dLgInlineFlex: true,
        dXlFlex: true,
        dXlInlineFlex: true,
        dXxlFlex: true,
        dXxlInlineFlex: true,
        flexRowReverse: true,
        flexColumn: true,
        flexColumnReverse: true,
        flexSmRow: true,
        flexSmRowReverse: true,
        flexSmColumn: true,
        flexSmColumnReverse: true,
        flexMdRow: true,
        flexMdRowReverse: true,
        flexMdColumn: true,
        flexMdColumnReverse: true,
        flexLgRow: true,
        flexLgRowReverse: true,
        flexLgColumn: true,
        flexLgColumnReverse: true,
        flexXlRow: true,
        flexXlRowReverse: true,
        flexXlColumn: true,
        flexXlColumnReverse: true,
        flexXxlRow: true,
        flexXxlRowReverse: true,
        flexXxlColumn: true,
        flexXxlColumnReverse: true,
        flexFill: true,
        flexSmFill: true,
        flexMdFill: true,
        flexLgFill: true,
        flexXlFill: true,
        flexXxlFill: true,
        flexGrow: 1,
        flexSmGrow: 1,
        flexMdGrow: 1,
        flexLgGrow: 1,
        flexXlGrow: 1,
        flexXxlGrow: 1,
        flexShrink: 1,
        flexSmShrink: 1,
        flexMdShrink: 1,
        flexLgShrink: 1,
        flexXlShrink: 1,
        flexXxlShrink: 1,
        flexNowrap: true,
        flexWrap: true,
        flexWrapReverse: true,
        flexSmWrap: true,
        flexSmNowrap: true,
        flexSmWrapReverse: true,
        flexMdWrap: true,
        flexMdNowrap: true,
        flexMdWrapReverse: true,
        flexLgWrap: true,
        flexLgNowrap: true,
        flexLgWrapReverse: true,
        flexXlWrap: true,
        flexXlNowrap: true,
        flexXlWrapReverse: true,
        flexXxlWrap: true,
        flexXxlNowrap: true,
        flexXxlWrapReverse: true
      },
    })
  })
  it('Содержимое html для app-flex-row ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
