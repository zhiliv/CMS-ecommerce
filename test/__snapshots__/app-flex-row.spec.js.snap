// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-flex-row.js Содержимое html для app-flex-row  1`] = `
<div
  class="d-flex d-inline-flex flex-row d-sm-flex d-sm-inline-flex d-md-flex d-md-inline-flex d-lg-flex d-lg-inline-flex d-xl-flex d-xl-inline-flex d-xxl-flex d-xxl-inline-flex flex-row-reverse flex-column flex-column-reverse flex-sm-row flex-sm-row-reverse flex-sm-column flex-sm-column-reverse flex-md-row flex-md-row-reverse flex-md-column flex-md-column-reverse flex-lg-row flex-lg-row-reverse flex-lg-column flex-lg-column-reverse flex-xl-row flex-xl-row-reverse flex-xl-column flex-xl-column-reverse flex-xxl-row flex-xxl-row-reverse flex-xxl-column flex-xxl-column-reverse flex-fill flex-sm-fill flex-md-fill flex-lg-fill flex-xl-fill flex-xxl-fill flex-grow-1 flex-sm-grow-1 flex-md-grow-1 flex-lg-grow-1 flex-xl-grow-1 flex-xxl-grow-1 flex-shrink-1 flex-sm-shrink-1 flex-md-shrink-1 flex-lg-shrink-1 flex-xl-shrink-1 flex-xxl-shrink-1 flex-nowrap flex-wrap flex-wrap-reverse flex-sm-wrap flex-sm-nowrap flex-sm-wrap-reverse flex-md-wrap flex-md-nowrap flex-md-wrap-reverse flex-lg-wrap flex-lg-nowrap flex-lg-wrap-reverse flex-xl-wrap flex-xl-nowrap flex-xl-wrap-reverse flex-xxl-wrap flex-xxl-nowrap flex-xxl-wrap-reverse simple"
  id="indent"
  style="background: white;"
/>
`;
